### PR TITLE
sec 2

### PR DESCRIPTION
**Pull request by kavindu0**

### CODE REVIEW

File: client/src/app/components/home/home.component.html [hunk 1]
```diff
@@ -111,6 +111,28 @@
   </div>
 </div>
 
+<div class="sec-3">
+  <div class="container">
+    <div class="d-md-flex align-items-center justify-content-between-w-100">
+      <div class="sec-3-cont">
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.html:117]: add section 3 content side

File: client/src/app/components/home/home.component.html [hunk 2]
```diff
@@ -111,6 +111,28 @@
   </div>
 </div>
 
+<div class="sec-3">
+  <div class="container">
+    <div class="d-md-flex align-items-center justify-content-between-w-100">
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.html:116]: create a div when up to medium size screens are flex and small size screens are block

File: client/src/app/components/home/home.component.html [hunk 3]
```diff
@@ -111,6 +111,28 @@
...
+          letters, as opposed to using</p>
+        <ul>
+          <li><i class="fa-solid fa-check"></i>More Than Just a Movie</li>
+          <li><i class="fa-solid fa-check"></i>Lights Out, Movie On</li>
+          <li><i class="fa-solid fa-check"></i>Enter a World of Imagination</li>
+        </ul>
+        <p class="p-1">It is a long established fact that a reader will distracted by the readable content of a page hen
+          looking at its layout.The point of using Lorem Ipsum</p>
+      </div>
+      <div class="sec-3-image">
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.html:129]: add image side in section 3

File: client/src/app/components/home/home.component.scss [hunk 4]
```diff
@@ -9,6 +9,22 @@
     transform: translateY(0);
   }
 }
+@keyframes slideInLeft {
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.scss:12]: create slide from left animation key frames

File: client/src/app/components/home/home.component.scss [hunk 5]
```diff
@@ -9,6 +9,22 @@
...
+  0% {
+    transform: translateX(-100%);
+  }
+  100% {
+    transform: translateX(0);
+  }
+}
+@keyframes slideInRight {
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.scss:20]: create slide from right side animation key frames

File: client/src/app/components/home/home.component.scss [hunk 6]
```diff
@@ -204,6 +220,50 @@ nav {
 
 }
 
+.sec-3{
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.scss:223]: section 3 styles are starts from here

File: client/src/app/components/home/home.component.scss [hunk 7]
```diff
@@ -204,6 +220,50 @@ nav {
...
+    align-items: center;
+    justify-content: center;
+    padding: 15px;
+    animation: slideInRight 1s ease-in-out;
+
+    img{
+      width: 100%;
+    }
+  }
+}
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.scss:265]: end of section 3 styles

File: client/src/app/components/home/home.component.scss [hunk 8]
```diff
@@ -267,5 +327,11 @@ nav {
       height: calc(100vh /2);
     }
   }
+
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.scss:330]: create section 3 mobile responsive

**Review by kavindu0 — COMMENTED** [review]
reviewed by kavi0

File: client/src/app/components/home/home.component.html (hunk 1 above)
Comment (kavicastelo) [reply client/src/app/components/home/home.component.html:117]: done

File: client/src/app/components/home/home.component.html (hunk 2 above)
Comment (kavicastelo) [reply client/src/app/components/home/home.component.html:116]: done

File: client/src/app/components/home/home.component.html (hunk 3 above)
Comment (kavicastelo) [reply client/src/app/components/home/home.component.html:129]: done

File: client/src/app/components/home/home.component.scss (hunk 4 above)
Comment (kavicastelo) [reply client/src/app/components/home/home.component.scss:12]: done

File: client/src/app/components/home/home.component.scss (hunk 5 above)
Comment (kavicastelo) [reply client/src/app/components/home/home.component.scss:20]: done

File: client/src/app/components/home/home.component.scss (hunk 6 above)
Comment (kavicastelo) [reply client/src/app/components/home/home.component.scss:223]: done

File: client/src/app/components/home/home.component.scss (hunk 7 above)
Comment (kavicastelo) [reply client/src/app/components/home/home.component.scss:265]: done

File: client/src/app/components/home/home.component.scss (hunk 8 above)
Comment (kavicastelo) [reply client/src/app/components/home/home.component.scss:330]: done

**Pull request merged**